### ISSUE TITLE
Option to exclude fields from export/writer

### ISSUE THREAD

**Issue by ar-jan** [sciunto-org/python-bibtexparser#195]
An option to omit specified fields from the BibTexWriter would be useful (unless I've missed some other way to achieve this, of course!). We could parse the bibtex database and remove fields, but I have a situation where I need a field to be in the bibtex database for sorting, but it should not be exported.

**Comment by omangin** [comment]
Indeed, that would be a valuable option since I believe it is a common use case. It should also be pretty straightforward to implement.

**Comment by MiWeiss** [comment]
Stale pull request for v1. Easily implementable for v2 through custom middleware (currently no builtin).
